### DISCUSSION TITLE
Why can't I chose existing existing provisioned evaluation group when creating a new alert?

### DISCUSSION THREAD

**Discussion by danfinn** [grafana/grafana#75253] [General]
I'm not able to create a new alert and point it to an existing evaluation group.  It shows the group in the dropdown and has the "provisioned" label next to it but I'm not able to select.  This makes for an annoying workflow when trying to create a new alert from an existing one.  I copy the alert but then I have to create a new bogus evaluation group, then I export the yaml for the alerts, then I have to go in manually and change the evaluation group back to the one I actually want.

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!
